src/main/webapp/style.css

```diff
@@ -50,6 +46,20 @@ body {
   background-color: #f3f3f3;
 }
 
+.btn-wide {
```

Comment:
since these share code you could put them together and differentiate the width

Reply:
Done